modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -86,12 +80,18 @@ function acquia_cms_common_install($is_syncing) {
       $config->setPluginsEnabled(['cee22bc3fe' => 'acquia_purge']);
     }
 
-    // Install the syslog module and purge sub-modules.
-    $module_installer->install([
+    // Contributed modules shipped by Acquia CMS.
+    $contributed_modules = [
+      'entity_clone',
+      'smart_trim',
+      'moderation_sidebar',
       'syslog',
       'purge_processor_cron',
       'purge_ui',
-    ]);
+    ];
+    // Install the syslog module and purge sub-modules.
+    // And other contributed modules provided by Acquia CMS.
+    $module_installer->install($contributed_modules);
```

Comment:
@apathak18, I believe it would be more appropriate to relocate this code from the .install file to a helper or service file and then invoke the method from there. What are your thoughts on this, @vishalkhode1?

Reply:
Are we repeating this logic somewhere?